pandas/tests/arrays/boolean/test_reduction.py

```diff
@@ -12,13 +12,16 @@ def data():
     )
 
 
+# .any(), .all() returns <NA> for [False, False], [True, True], GH-33253.
 @pytest.mark.parametrize(
     "values, exp_any, exp_all, exp_any_noskip, exp_all_noskip",
     [
         ([True, pd.NA], True, True, True, pd.NA),
         ([False, pd.NA], False, False, pd.NA, False),
         ([pd.NA], False, True, pd.NA, pd.NA),
         ([], False, True, False, True),
+        ([True, True], True, True, True, True),
```

Comment:
```suggestion
@pytest.mark.parametrize(
    "values, exp_any, exp_all, exp_any_noskip, exp_all_noskip",
    [
        ([True, pd.NA], True, True, True, pd.NA),
        ([False, pd.NA], False, False, pd.NA, False),
        ([pd.NA], False, True, pd.NA, pd.NA),
        ([], False, True, False, True),
        # GH-33253: all True / all False values buggy with skipna=False
        ([True, True], True, True, True, True),
```

(the diff looks huge, but I just moved the comment down below to just above the added line)